datadog_lambda/wrapper.py

```diff
@@ -72,6 +89,13 @@ def __call__(self, event, context):
         self._before(event, context)
         try:
             return self.func(event, context)
+        except Exception:
+            lambda_metric(
```

Comment:
Although this is a oneliner, I would make it a function in metric.py, to keep the wrapper focus on code flow rather than details.